samples/feature_group/README.md

```diff
@@ -0,0 +1,108 @@
+# Feature Group Sample
+
+This sample demonstrates how to create a feature group using the Amazon AWS Controllers for Kubernetes (ACK) service controller for Amazon SageMaker.
+
+## Prerequisites
+
+This sample assumes that you have completed the [common prerequisites](https://github.com/aws-controllers-k8s/sagemaker-controller/blob/main/samples/README.md).
+
+## Create an S3 bucket:
+
+Since we are using the offline store in this example, you need to set up an s3 bucket. [Here are directions](https://docs.aws.amazon.com/AmazonS3/latest/userguide/create-bucket-overview.html) to set up your s3 bucket through the S3 Console, AWS SDK, or AWS CLI.
+
+## Adding required policies to your IAM role:
+
+First create an Amazon SageMaker execution role, and then [here are directions](https://docs.aws.amazon.com/sagemaker/latest/dg/feature-store-adding-policies.html) to give your SageMaker Execution Role SageMakerFeatureStoreAccess. Make sure that your role has access to your s3 bucket, this can be done by giving it AmazonS3FullAccess.
```

Comment:
This section is not required 

in the pre-req we already show them how to create IAM role and give SagemakerFullAccess and S3FullAccess